setup.sh

```diff
@@ -1,28 +1,31 @@
 #!/bin/bash -e
 
 #####################################################
-#
 # Set up the Membership application
-#
 #####################################################
 
-echo " ____ ____ ____ ____ ____ ____ ____ ____ ____ ____ ";
-echo "||m |||e |||m |||b |||e |||r |||s |||h |||i |||p ||";
-echo "||__|||__|||__|||__|||__|||__|||__|||__|||__|||__||";
-echo "|/__\|/__\|/__\|/__\|/__\|/__\|/__\|/__\|/__\|/__\|";
-
-printf "\n\r"
-
-echo "  ^    ^    ^    ^    ^    ^    ^    ^  "
-echo " /f\  /r\  /o\  /n\  /t\  /e\  /n\  /d\ "
-echo "<___><___><___><___><___><___><___><___>"
-
```

Comment:
In fact, the only way I'll accept this being removed is if you replace it with something even more ridiculous.